lib/WUI/sntp/sntp_client.h

```diff
@@ -5,8 +5,10 @@
 extern "C" {
 #endif
 
-void sntp_client_init(void);
-void sntp_client_step(void);
+void sntp_client_static_init(const char *ntp_address);
+void sntp_client_dhcp_init(const char *ntp_address);
```

Comment:
These inits probably aren't called from the outside, so maybe they should be static / anonymous namespace and not exported.

Reply:
yep.

Reply:
sntp_client_static_init is used in wui.cpp, thus cannot be static. But I made sntp_client_dhcp_init static.